### PR TITLE
Autofocus network error

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/23838

## Description
Improved handling for connection error

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2_test.py [hunk 1]
```diff
@@ -119,3 +120,24 @@ def test_calculate_dbot_score_file():
...
     score = calculate_dbot_score(raw_indicator, 'File')
     assert score == 3
+
+
+def test_connection_error(mocker):
+
+    import AutofocusV2
+    import requests
+    import requests_mock
+    import sys
```
Comment (ronykoz) [inline Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2_test.py:130]: why are these here?

**Comment by ronykoz** [comment]
let's change the RN for something like: "Better error message for connection errors"

File: Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2_test.py (hunk 1 above)
Comment (barchen1) [reply Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2_test.py]: fixed
